custom_components/tahoma/cover.py

```diff
@@ -40,6 +40,9 @@
 COMMAND_SET_CLOSURE_AND_LINEAR_SPEED = "setClosureAndLinearSpeed"
 COMMAND_SET_DEPLOYMENT = "setDeployment"
 COMMAND_SET_ORIENTATION = "setOrientation"
+# io:DiscreteGateOpenerIOComponent
+COMMAND_SET_PEDESTRIAN_POSITION = "setPedestrianPosition"
```

Comment:
Well actually it’s not really correct: 
```json
    {
      "commandName": "setPedestrianPosition",
      "nparams": 0
    },
```
It’s a setPosition like we are used to. This one does not take any parameters.